README.md

```diff
@@ -240,6 +240,10 @@ card.provision(
 
 ***
 
+# Out-of-Band (OOB) Authentication
+
+Our CheckoutCardManagement-Android SDK is compatible with our Out-of-Band (OOB) Authentication solution, which is covered through our [OOB-Android SDK]([https://www.checkout.com/docs/payments/regulation-support/pci-compliance], and you can read integration details on that page. Checkout.com's OOB SDK enables you to seamlessly authenticate your cardholders within your Android app.
+
```

Comment:
Few suggestions:
1. Shall we add OOB Android SDK repo link for OOB-Android-SDK word?
2. Currently, [OOB-Android SDK] is not linkable work looks like as attached screenshot.
![Screenshot 2024-04-03 at 11 38 11](https://github.com/checkout/CheckoutCardManagement-Android/assets/114917119/977603f0-2ad9-43cd-ab47-044cf4c6e2d5)
We can have something like below to make it work:
`[OOB-Android SDK](https://www.checkout.com/docs/payments/regulation-support/pci-compliance)`

Reply:
Oops, my bad. There is a typo here, let me re-do

Reply:
Addressed changes